sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Infrastructure/EventProcessorManager.cs

```diff
@@ -331,9 +332,6 @@ private class ShortWaitTimeMock : EventProcessorClient
             public static readonly TimeSpan ShortLoadBalanceUpdate = TimeSpan.FromSeconds(1);
             public static readonly TimeSpan ShortOwnershipExpiration = TimeSpan.FromSeconds(3);
 
-            internal override TimeSpan LoadBalanceUpdate => ShortLoadBalanceUpdate;
-            internal override TimeSpan OwnershipExpiration => ShortOwnershipExpiration;
-
```

Comment:
I'm aware we are probably going to dispose of this file along with the Event Processor Live Tests, but, for the time being, could you keep at least the short load balance update value so it doesn't interfere with the current tests that depend on it? If I'm not mistaken, the `OwnershipExpiration` currently is not necessary for the tests.

You may need to create a `PartitionLoadBalancer` instance in the `EventProcessorManager` constructor to pass to its processors.

Reply:
Sure thing - let me know if you have any concerns with the approach I took,